docs/execution-providers/STVM-ExecutionProvider.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Standalone TVM (STVM)
+description: Instructions to execute ONNX Runtime with the Standalone TVM (STVM) execution provider
+parent: Execution Providers
+nav_order: 15
+redirect_from: /docs/reference/execution-providers/STVM-ExecutionProvider
+---
+
+# Standalone TVM (STVM) Execution Provider
+{: .no_toc }
+
+STVM is an execution provider for ONNX Runtime that is built on top of Apache TVM. It enables ONNX Runtime users to leverage Apache TVM model optimizations.
+STVM EP is currently in "Preview". It's been tested to work on a handful of models on Linux, but not on Windows or MacOS.
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Build
```

Comment:
The current pattern is to have the build instructions for EPs on the build page. I think it's fine for them to be here. We can decide later which is the better pattern.

Reply:
We can definitely work on a follow up PR that moves the build instructions into the build page to match the other EP guidelines.